examples/huggingface/neural_speed/run_accuracy.py

```diff
@@ -25,22 +25,22 @@
     args = parser.parse_args()
     print(args)
     model_args=f'pretrained="{args.model_name}",dtype=float32,trust_remote_code=True'
-    if args.use_gptq:
-        model_args += ",use_gptq=True"
+
     if args.model_format == "neural_speed":
-        results = evaluate(
-            model="hf-causal",
-            model_args=model_args,
-            batch_size=args.batch_size,
-            tasks=[f"{args.tasks}"],
-            model_format=f"{args.model_format}"
-        )
+        eval_args = LMEvalParser(model="hf", 
```

Comment:
It doesn't go inside neural_speed

Reply:
here is referenece, please make it run successfully, please call me if something needed. :)
```
    model_args=f'pretrained="{args.model_name}",dtype=float32,trust_remote_code=True'

    if args.model_format == "neural_speed":
        model_args += f",model_format=neural_speed"
```